### ISSUE TITLE
add `description` to every parameter in `action.yml`

### ISSUE THREAD

**Issue by yegor256** [zerocracy/judges-action#93]
Let's add descriptions, as it's done here: https://github.com/JamesIves/github-pages-deploy-action/blob/dev/action.yml

**Comment by artur-intech** [comment]
"as it's done here" sounds vague. Can you clarify? Currently every parameter has corresponding description and at least for a new person like me it seems OKish.

**Comment by yegor256** [comment]
@artur-intech agree, thanks. BTW, read this please: https://www.yegor256.com/2024/04/01/ping-me-please.html

**Comment by artur-intech** [comment]
I tend to mention people but you are the author so I assumed you are always notified.
